pennylane/tape/operation_recorder.py

```diff
@@ -50,7 +50,6 @@ def __init__(
         prep=None,
         shots=None,
         name=None,
-        do_queue=False,

```

Comment:
I'm not familiar with this recorder, are we fine just removing this kwarg?

Reply:
Hm, I don't know either. The argument is ignored by `OperationRecorder` and the class has no children. Also, whenever `OperationRecorder` is instantiated in PennyLane, no arguments are passed. I'd assume if all tests pass with this change, it should be fine.